packages/did-ethr-resolver/src/implementations/ewPrivateKeySigner.ts

```diff
@@ -0,0 +1,31 @@
+import { EwSigner } from './ewSigner';
+import { providers, Wallet } from 'ethers';
+
+export class EwPrivateKeySigner extends EwSigner {
+  private _wallet: Wallet;
+
+  private _privateKey: string;
+
+  private _provider: providers.Provider;
```

Comment:
I think it's best to mark as readonly when possible
```suggestion
  private readonly _wallet: Wallet;

  private readonly _privateKey: string;

  private readonly _provider: providers.Provider;
```